test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java

```diff
@@ -21,210 +21,108 @@
  * questions.
  */
 
-/**
+import java.awt.Frame;
+import java.awt.print.PageFormat;
+import java.awt.print.PrinterJob;
+import java.lang.reflect.InvocationTargetException;
+
+import javax.print.PrintService;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.SwingUtilities;
+
+/*
  * @test
  * @bug 4937672 5100706 6252456
  * @key printer
- * @run main/othervm/manual -Djava.security.manager=allow SecurityDialogTest
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual/othervm -Djava.security.manager=allow SecurityDialogTest
  */
```

Comment:
Line spacing here would be nice.

Reply:
@TejeshR13 What do you mean? What kind of line spacing would be nice here?

Reply:
Line spacing between JTREG tag and class.

Reply:
It's not needed… but most people add it. And we cannot agree on *a common style*, unfortunately.

These jtreg tags are akin to javadoc comment (but they're not javadoc comment, and javadoc syntax `/**` with double-asterisk should not be used), and in most cases the javadoc comment is not separated with a blank line from its class.

The last line of the jtreg tags comment is nearly empty by itself.